mixserv/src/main/java/hivemall/mix/server/MixServerHandler.java

```diff
@@ -123,7 +133,13 @@ private void mix(final ChannelHandlerContext ctx, final MixMessage requestMsg, f
         final int deltaUpdates = requestMsg.getDeltaUpdates();
         final boolean cancelRequest = requestMsg.isCancelRequest();
 
-        MixMessage responseMsg = null;
+        if (deltaUpdates <= 0)
```

Comment:
Please use this modification only to mix().
BTW, put `{` and `}` for every conditional branch.